### ISSUE TITLE
spread the load for the api calls

### ISSUE THREAD

**Issue by kisst** [kisst/CFN-Drift-Reports#1]
in case there are a lot of stacks, aws can limit the api-calls, which can cause, the detect not to run, with an SQS, and queue processing, this can be fixed, which also should provide an option for retry, as right now it's fire and forget only

**Comment by AndreyZaslavskiyWMG** [comment]
Did you experienced that kind of problems? And how much stacks if limit for current solution like "fire-and-forget" ?

**Comment by kisst** [comment]
Yes I had cases where I got throttled, but at the same time that was for accounts where I had to already request CFN limit increase as there was more then 200 Stacks in each region in use. In general for most use-case this already works, but in a local draft branch I have most of the pieces for the fancy DynamoDB / Queue based solution, just haven't had time to complete it, but happy to push it to GH if want to pick it up.
